spec/jobs/spotlight/add_uploads_from_csv_spec.rb

```diff
@@ -26,11 +26,12 @@
   end
 
   it 'creates uploaded resources for each row of data' do
-    expect(Spotlight::Resources::Upload).to receive(:new).with(hash_including(remote_url_url: 'x')).and_return(resource_x)
-    expect(Spotlight::Resources::Upload).to receive(:new).with(hash_including(remote_url_url: 'y')).and_return(resource_y)
+    upload = FactoryGirl.create(:uploaded_resource)
+    expect(Spotlight::Resources::Upload).to receive(:new).at_least(:once).and_return(upload)
 
-    expect(resource_x).to receive(:save_and_index)
-    expect(resource_y).to receive(:save_and_index)
+    expect(upload).to receive(:build_upload).with(remote_image_url: 'x').and_call_original
```

Comment:
Hoping `.and_call_original` will keep this from passing in the future if associations change underneath.